### ISSUE TITLE
[HOLD for payment 2022-11-04] [$250] Bug: No focus indicator at edit composer reported by @Puneet-here

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#11728]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Go to any chat > Send a message
2. Edit

## Expected Result:
There should be a focus indicator (blue border) like our all other fields

## Actual Result:
There's no focus indicator

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.12-3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/195197826-c68a1a29-5a90-4259-a20a-9f91992dad9b.mp4

<img width="1067" alt="Screenshot 2022-10-11 at 8 13 33 PM" src="https://user-images.githubusercontent.com/43996225/195197840-1f201bb3-4d88-4a98-9451-2eb8b389bd5f.png">

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @Puneet-here
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1665499439983919

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Puneet-here** [comment]
### Proposal

We need to add a state "isFocused" we will set it to true at `onFocus` and false at `onBlur` prop of Composer 
https://github.com/Expensify/App/blob/757fb5e43398b3476ed75889fd689e2bdf043777/src/pages/home/report/ReportActionItemMessageEdit.js#L203-L212
we need to use isFocused like below to change border color
```this.state.isFocused ? styles.chatItemComposeBoxFocusedColor : styles.chatItemComposeBoxColor```

https://github.com/Expensify/App/blob/2f7f5ead742b7e11dd0102a9ff031d848b7ff97f/src/pages/home/report/ReportActionItemMessageEdit.js#L191

**Comment by pecanoro** [comment]
I think this has worked this way forever so I am not sure if it's worth fixing it, but I am going to reach out to the rest of the team and I will come back to export it or close it.

**Comment by pecanoro** [comment]
Ok, we agreed we should fix it, so assigning external1

**Comment by ctkochan22** [comment]
Looks good @jliexpensify lets hire @Puneet-here

**Comment by jayeshmangwani** [comment]
@kavimuru I [reported](https://expensify.slack.com/archives/C01GTK53T8Q/p1656619624743589) this 3 months ago, can you please check this once?

**Comment by jliexpensify** [comment]
@parasharrajat - I think you could make a call here, but it feels like @jayeshmangwani should get the Reporting fee, and @Puneet-here should get the PR fee.

**Comment by parasharrajat** [comment]
Seems so.

**Comment by Puneet-here** [comment]
Yes, It's fine by me, since it was a dupe.

**Comment by jliexpensify** [comment]
Posted:

- Internal https://www.upwork.com/ab/applicants/1582854050876055552/job-details
- External https://www.upwork.com/jobs/~01fa637bc66f289807

Everyone has been invited, thanks!

**Comment by Puneet-here** [comment]
Sorry for the delay. I was waiting for my another PR to get merged so that we don't get conflicts. I have raised the PR now.

**Comment by ctkochan22** [comment]
Not overdue

**Comment by jliexpensify** [comment]
Good to pay, please accept your offers. @parasharrajat also, please accept the invite, cheers!

**Comment by jliexpensify** [comment]
Paid Puneet and Jayesh (+ contracts ended), will close once Rajat has accepted and is paid.

**Comment by parasharrajat** [comment]
@jliexpensify Done.

**Comment by jliexpensify** [comment]
All done and job closed. Thanks everyone!
